src/components/common/seo/CanonicalUrl.tsx

```diff
@@ -0,0 +1,26 @@
+import { Helmet } from "react-helmet-async";
+import { useLocation } from "react-router-dom";
+
+// Configuration
+const SITE_URL = "https://portal-beta.aodn.org.au";
```

Comment:
How about other env? Can be be like setup in the .env file?

Also if just a <link> need so complicated? 

Can code it like another google or newrelic like import?